### ISSUE TITLE
Error in /nirs(i)/metaDataTags/LengthUnit

### ISSUE THREAD

**Issue by Edouard2laire** [fNIRS/snirf-samples#5]
Hi, 

I think there is an error in the two exemples in basic folder. In both exemples,  /nirs(i)/metaDataTags/LengthUnit are set to 'mm', or I think it should be cm here. 

In basic probe example, detector are in  each corner of a square :  
(x=0,y=0); (x=4;y=0);( x=0;y=4) and (x=4,y=4) with the source in the middle at  (x=2, y=2). 

So the source-detector distance is 2.8. Which is coherent only if LengthUnit is in centimeter. Same issue occurs in neuro_run01.snirf

Best regards, 
Edouard

**Comment by fangq** [comment]
@Edouard2laire, I was just about to fix this, but recognized some issues with the sample files that was previously created from `snirf_homer3`, see

https://github.com/fNIRS/snirf_homer3/issues/4

I will let @jayd1860 to fix the snirf parser to generate the correct dimension, in the meantime, I will recreate the sample data using my easyh5 toolbox which does not have this issue.

stay tuned.

**Comment by fangq** [comment]
@Edouard2laire, the unit is issue is now fixed. I also fixed the array dimensions in easyh5 (for snirf_homer3 parser, @jayd1860 is working on a fix) and regenerated all sample data files, also including their jsnirf versions (.jnirs, .bnirs) for environments where hdf5 is not supported (like matlab older than R2011 and octave).

The updated data files also included the updated data fields as described in this patch
https://github.com/fNIRS/snirf/issues/41#issuecomment-629675313

closing this ticket for now.
